test/integration/golden_response/staging/get_triples/place_type.json

```diff
@@ -3041,7 +3041,7 @@
       "objectTypes": [
         "Class"
       ],
-      "provenanceId": "dc/5n63hr1"
+      "provenanceId": "dc/sm3m2w3"
```

Comment:
the provenance is different for current and svobs mode. I wonder shall we make the prov stable, or fork the test? (lean towards stable prov) @pradh

Reply:
Provenance is not stable even on current prod, so I doubt if this is popobs vs. svobs issue... its just a long-standing thing that we need to fix.

Reply:
Yea, the reason is if they are not the same, then need to create another golden file for svobs, then it's hard to compare the diff.